tests/e2e/performance/io_workload/test_small_file_workload.py

```diff
@@ -546,13 +529,18 @@ def teardown(self):
         log.info("cleanup the environment")
         if isinstance(self.es, ElasticSearch):
             self.es.cleanup()
-        self.operator.cleanup()
+        try:
+            self.operator.cleanup()
+        except Exception:
+            # nothig to do, the benchmark-operator did not deployed. this is for
+            # the results collecting and pushing results into the dashboard
+            pass
```

Comment:
Not sure if I am getting it correct here, if we are not checking for operator then better to remove and with 'pass' in exception, in-case of failures in try block exception will just pass without handling it.

Reply:
in this case , i am trying to use the operator.cleanup() , if it is exist it will run (there is no exception raising within the function), otherwize we will catch the exception (of not exist object) and in that case, we just continue without doing anything regarding the operator. the operator object is benchmark-operator object, which not all tests in our suite use.